acceptance_tests/features/steps/rh_ui_cookies.py

```diff
@@ -0,0 +1,110 @@
+from behave import step
+
+import json
+
+from acceptance_tests.utilities.test_case_helper import test_helper
+from config import Config
+
+RETURN = "\ue006"
+
+
+@step("the cookies page is displayed")
+def display_uac_cookies_page(context):
+    context.browser.visit(f"{Config.RH_UI_URL}en/cookies")
+
+
+# Cookies banner checks and interactions
+
+
+@step("the cookies banner is displayed")
+def display_cookies_banner(context):
+    context.browser.cookies.delete_all()  # clear cookies to ensure that the banner is displayed
+    context.browser.find_by_id("ons-cookies-banner")
+
+
+@step("the user accepts the cookies on the cookies banner")
+def accept_cookies_on_cookies_banner(context):
+    xpath_string = '//button[contains(@class,"ons-js-accept-cookies")]'
+    context.browser.find_by_xpath(xpath_string).click()
+
+
+@step("the user rejects the cookies on the cookies banner")
+def reject_cookies_on_cookies_banner(context):
+    xpath_string = '//button[contains(@class,"ons-js-reject-cookies")]'
+    context.browser.find_by_xpath(xpath_string).click()
+
+
+@step("the 'View cookies' hyperlink points to {expected_path}")
+def click_view_cookies_link_on_banner(context, expected_path):
+    xpath_string = '//a[@class="ons-cookies-banner__link" and text()="View cookies"]'
+    link = context.browser.find_by_xpath(xpath_string).first
+    test_helper.assertEqual(link["href"], f"{Config.RH_UI_URL}{expected_path}")
+
+
+@step("the 'cookies' hyperlink on the cookies banner points to {expected_path}")
+def click_cookies_hyperlinked_text_on_banner(context, expected_path):
+    xpath_string = '//div[@class="ons-cookies-banner__statement"]/a[text()=" cookies."]'
+    link = context.browser.find_by_xpath(xpath_string)
+    test_helper.assertEqual(link["href"], f"{Config.RH_UI_URL}{expected_path}")
+
+
+@step('the "change cookie preferences" hyperlink text points to {expected_path}')
+def click_change_cookie_preferences_link_on_banner(context, expected_path):
+    xpath_string = '//span[@class="ons-cookies-banner__preferences-text"]/a[text()=" change your cookie preferences"]'
+    link = context.browser.find_by_xpath(xpath_string)
+    test_helper.assertEqual(link["href"], f"{Config.RH_UI_URL}{expected_path}")
+
+
+# Cookies page radio interactions
+
+
+@step("the user sets the selection under {para_title} to {cookie_selection}")
+def change_cookies_selection(context, para_title, cookie_selection):
+    match para_title:
```

Comment:
Could add a default case for this match to fail out immediately if it doesn't match the title, to be clear it's an issue with the page content, otherwise it would look like a test code issue when `name_attribute_value` isn't assigned.

Reply:
Perhaps would be better to fail the test directly, since the content of the page isn't matching the test expectation, and include the unexpected value? e.g.
```python
test_helper.fail(f"Found unexpected cookies paragraph title: {para_title}")
```